hedera-node/src/main/java/com/hedera/services/store/tokens/BaseTokenStore.java

```diff
@@ -463,6 +463,27 @@ private ResponseCodeEnum tryAdjustment(AccountID aId, TokenID tId, long adjustme
 		return OK;
 	}
 
+	protected ResponseCodeEnum tryAdjustment(AccountID senderAId, AccountID receiverAId, TokenID tId, long serialNumber) {
```

Comment:
I think that we can reuse the logic for checking IS FROZEN, IS KYC with the other `tryAdjustment` method